src/app/pages/OnDemandAudioPage/index.styles.ts

```diff
@@ -1,27 +1,77 @@
 import { css, Theme } from '@emotion/react';
-import { GROUP_2_MAX_WIDTH_BP } from '#app/components/ThemeProvider/mediaQueries';
+import pixelsToRem from '../../utilities/pixelsToRem';
 
-export const GROUP_1_FROM_360PX_AND_GROUP_2 = `@media (min-width: 22.5rem) and (max-width: ${GROUP_2_MAX_WIDTH_BP}rem)`;
+const showOnDemandImage = '22.4375rem';
 
 export default {
-  wrapper: ({ mq, spacings }: Theme) =>
+  grid: ({ mq, gridWidths }: Theme) =>
     css({
+      maxWidth: `${pixelsToRem(gridWidths[1008])}rem`,
+      margin: '0 auto',
+      display: 'grid',
+      gridTemplateColumns: 'repeat(6, 1fr)',
+      columnGap: '0.5rem',
```

Comment:
I am wondering if we want to use the `spacings` from the Theme provider here (and in other places in this file), or is that less readable?

 ```suggestion
       columnGap: `${spacings.FULL}rem`,
```

Reply:
Good point!